Engine/csound_orc_semantics.c

```diff
@@ -1467,8 +1468,8 @@ void add_arg(CSOUND* csound, char* varName, char* annotation, TYPE_TABLE* typeTa
   var = csoundFindVariableWithName(csound, pool, varName);
   if (var == NULL) {
     if (annotation != NULL) {
-      type = csoundGetTypeWithVarTypeName(csound->typePool, annotation);
-      typeArg = type;
+      type = (CS_TYPE *) csoundGetTypeWithVarTypeName(csound->typePool, annotation);
```

Comment:
Rather than cast to (CS_TYPE *) everywhere, shouldn't type be changed to const CS_TYPE* ?

Reply:
(and same comment for all places csoundGetTypeWithVarTypeName is being used)

Reply:
I tried that but there was a point where it was awkward. I can try again.

Reply:
I see why it was awkward, I had to enforce everywhere. That's the right thing though:
9aea5349a0c7ed51910a88d3ffd82853adce4671 
bd70b3254d11515e9c40cc514ad9a99d6364239f